examples/env/baseline.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
```

Comment:
Perhaps can rename this `examples/env/baseline.py` file on creating and registering an env as `examples/env/figure_eight_v0_env.py`